elide-standalone/src/main/java/com/yahoo/elide/standalone/Util.java

```
@@ -53,28 +64,61 @@ public static EntityManagerFactory getEntityManagerFactory(String modelPackageNa
         }
 
-        PersistenceUnitInfo persistenceUnitInfo = new PersistenceUnitInfoImpl("elide-stand-alone",
-                combineModelEntities(modelPackageName, includeAsyncModel), options);
+        dynamicEntityCompiler = new ElideDynamicEntityCompiler(dynamicConfigPath);
+        if (includeDynamicModel) {
+            dynamicEntityCompiler.compile();
+            Collection<ClassLoader> classLoaders = new ArrayList<>();
+            classLoaders.add(dynamicEntityCompiler.getClassLoader());
+            options.put(AvailableSettings.CLASSLOADERS, classLoaders);
+        }
+        PersistenceUnitInfo persistenceUnitInfo = null;
+        try {
+
+            ElideDynamicEntityCompiler.bindClasses = new HashSet<>();
+            // add dynamic generated classes
+            ElideDynamicEntityCompiler.bindClasses.addAll(populateBindClasses(dynamicEntityCompiler, Entity.class));
+            ElideDynamicEntityCompiler.bindClasses.addAll(populateBindClasses(dynamicEntityCompiler, FromTable.class));
+            ElideDynamicEntityCompiler.bindClasses.addAll(populateBindClasses(dynamicEntityCompiler,
+                    FromSubquery.class));
+            // add classes
+            ElideDynamicEntityCompiler.bindClasses.addAll(ClassScanner.getAnnotatedClasses(Entity.class));
+            ElideDynamicEntityCompiler.bindClasses.addAll(ClassScanner.getAnnotatedClasses(FromTable.class));
+            ElideDynamicEntityCompiler.bindClasses.addAll(ClassScanner.getAnnotatedClasses(FromSubquery.class));
+
+            persistenceUnitInfo = new PersistenceUnitInfoImpl("elide-stand-alone",
+                    combineModelEntities(dynamicEntityCompiler, modelPackageName,
+                            includeAsyncModel, includeDynamicModel),
+                    options,
+                    dynamicEntityCompiler.getClassLoader());
 
+        } catch (ClassNotFoundException e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
```

We should probably not catch the exception and let the error propagate.